.circleci/local_publish_helpers.sh

```diff
@@ -71,7 +71,22 @@ function generatePkgCli {
   # Build pkg cli
   cp package.json ../build/node_modules/package.json
   if [[ "$CIRCLE_BRANCH" == "release" ]] || [[ "$CIRCLE_BRANCH" =~ ^run-e2e-with-rc\/.* ]] || [[ "$CIRCLE_BRANCH" =~ ^release_rc\/.* ]] || [[ "$CIRCLE_BRANCH" =~ ^tagged-release ]]; then
-    npx pkg -t node14-macos-x64,node14-linux-x64,node14-linux-arm64,node14-win-x64 ../build/node_modules --out-path ../out
+    # This will generate a file 'amplify-pkg' in the '../tmp-out' directory for our arm64 binary.
+    # The file name does not include the generated <platform>-<arch> suffixes because there is a single target.
+    npx pkg --no-bytecode -t node14-linux-arm64 ../build/node_modules --out-path ../tmp-out
+
+    # This will generate files 'amplify-pkg-<platform>' in the '../out' directory for our x64 binaries.
+    # The <platform> tag is included in the output file names because there are different
+    # platforms (macos/linux/win) being built in the same pkg call.
+    npx pkg -t node14-macos-x64,node14-linux-x64,node14-win-x64 ../build/node_modules --out-path ../out
+
+    # this renames the x64 files so that they have the appropriate suffix 'x64'
+    mv ../out/amplify-pkg-macos ../out/amplify-pkg-macos-x64
+    mv ../out/amplify-pkg-linux ../out/amplify-pkg-linux-x64
+    mv ../out/amplify-pkg-win.exe ../out/amplify-pkg-win-x64.exe
+
+    # this renames the arm64 file so that it has the appropriate suffix 'linux-arm64'
+    mv ../tmp-out/amplify-pkg ../out/amplify-pkg-linux-arm64
```

Comment:
maybe we should just call npx pkg 4 times and use `-o` aka `--output` parameter ?
i.e.
`npx pkg -t node14-macos-x64 ../build/node_modules --out-path ../out --output amplify-pkg-macos-x64` and so on ?

Reply:
will experiment with --o; I had issues with it misinterpreting ../out/amplify-pkg-linux-x64 as a directory but there were other issues I was working through as well. let me try again

Reply:
running each one individually adds about 5 minutes: 

current (11 min) https://app.circleci.com/pipelines/github/aws-amplify/amplify-cli/11589/workflows/b3912cf6-9c09-47aa-a6b6-ab844920c071/jobs/412533

after (16 min) https://app.circleci.com/pipelines/github/aws-amplify/amplify-cli/11594/workflows/94d829a6-a835-453a-813d-b1b141a4e768/jobs/412551

Reply:
I am ok to be slower here, for readability; this will be replaced soon anyway. PR updated!

Reply:
This might help https://stackoverflow.com/questions/3004811/how-do-you-run-multiple-programs-in-parallel-from-a-bash-script .

Reply:
yet one more option to explore is that `-o` seems to accept name template, so there should be a way to force it to include arch component.